kokoro/scripts/test/go_test_for_containers.sh

```diff
@@ -107,6 +107,9 @@ if [[ "${SHORT:-false}" == "true" ]]; then
   args+=( "-test.short" )
 fi
 
+# Avoids "fatal: detected dubious ownership in repository" errors on Kokoro containers.
+git config --global --add safe.directory ...
+
```

Comment:
I would have put this up near the top of the file. Certainly it feels weird being in between "setting up args" and "using args for the actual go test command".

Reply:
Fair